Source/CesiumRuntime/Public/CesiumMetadataUtilityBlueprintLibrary.h

```diff
@@ -71,35 +68,4 @@ class CESIUMRUNTIME_API UCesiumMetadataUtilityBlueprintLibrary
   static TMap<FString, FString> GetMetadataValuesAsStringForFace(
       const UPrimitiveComponent* component,
       int64 faceID);
-
-  /**
-   * Gets the feature ID associated with a given face for a feature id
-   * attribute.
-   */
-  UFUNCTION(
-      BlueprintCallable,
-      BlueprintPure,
-      Category = "Cesium|Metadata|Utility")
-  static int64 GetFeatureIDFromFaceID(
```

Comment:
Again, this tries to act on a `FCesiumMetadataPrimitive` which no longer manages the feature IDs. But this should probably be readded for backwards compatibility.